crates/autopilot/src/database/orders.rs

```diff
@@ -1,25 +1,17 @@
 use super::Postgres;
 use anyhow::Result;
 use chrono::{Duration, Utc};
-use ethcontract::U256;
+use database::byte_array::ByteArray;
 use futures::{StreamExt, TryStreamExt};
 use model::{
     order::{Order, OrderUid},
     time::now_in_epoch_seconds,
 };
 use number_conversions::u256_to_big_decimal;
-use shared::db_order_conversions::full_order_into_model_order;
-
-/// New fee data to update a limit order with.
-pub struct FeeUpdate {
```

Comment:
What has to happen is pretty clear to me but I'm not sure how the interface is supposed to look like in light of the recent design discussions. `Quote` pretty accurately contains all the data that is needed here so I preferred to use it instead of copying all the required fields into a new type or `FeeUpdate`.
However maybe this code is not supposed to rely on a type coming from `shared::order_quoting::Quote` here. cc @ennmichael